patterns/newtype.md

```diff
@@ -1,8 +1,11 @@
 # Newtype
 
-Rust has strong static types. This can be very different than what you are used to if you are coming from a loosely-typed language. Don't worry, though. Once you get used to them, you'll find the types actually make your life easier. Why? Because you are making implicit assumptions explicit.
+What if in some cases we want a type to behave similar to another type but
+still different? What if we want to enforce some behavior at compile time
+which using only type aliases is not enough? For example, have a custom
+`Display` implementation for `String` for security reasons (password).
```

Comment:
```suggestion
What if in some cases we want a type to behave similar to another type or 
enforce some behaviour at compile time where using only type aliases would 
not be enough? 

For example, if you want to create a custom `Display` implementation for `String` 
due to security considerations (e.g. passwords).
```

Reply:
The separation looks good but the word "you" does not look that good for me. Either way I just applied it.